modules/test/playwright/helpers/ApiHelpers.ts

```diff
@@ -363,6 +363,9 @@ export class DataApiHelpers extends ApiHelpers {
 			else if (item.type === 'apiApplication') {
 				await this.apiBuilder.deleteApiApplication(item.id);
 			}
+			else if (item.type === 'apiEndpoint') {
```

Comment:
I think the API Endpoint should be deleted once the api application is deleted, at the same time. Could you please check if the API Application is enough? In that case, please remove the code. If not, let me know and I will review again the Pr